pkg/cloud/services/loadbalancer/loadbalancer.go

```diff
@@ -267,7 +267,7 @@ func (s *Service) getOrUpdateAllowedCIDRS(openStackCluster *infrav1.OpenStackClu
 	listener.AllowedCIDRs = capostrings.Unique(listener.AllowedCIDRs)
 
 	if !reflect.DeepEqual(allowedCIDRs, listener.AllowedCIDRs) {
-		s.scope.Logger().Info("CIDRs do not match, start to update listener", "expected CIDRs", allowedCIDRs, "load balancer existing CIDR", listener.AllowedCIDRs)
+		s.scope.Logger().Info("CIDRs do not match, start to update listener", "expected CIDRs", allowedCIDRs, "existing CIDR", listener.AllowedCIDRs)
```

Comment:
This is basically a trace level log message. When we get around to it, I'm pretty sure we'll put this at `V(5)`. However...

I'm not 100% sure, but I don't think we want spaces in parameter names. Perhaps `expected` and `current`?

We also probably want `..., updating listener`.

And lastly: are you able to find out how `allowedCIDRs` renders in this log message in practise? I wonder if we should explicitly format it somehow. However, don't spend much time on this given that it's most likely a trace level log.